langchain/src/stores/message/utils.ts

```diff
@@ -6,10 +6,17 @@ import {
   SystemChatMessage,
 } from "../../schema/index.js";
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type AdditionalKwargs = Record<string, any>;
+export interface StoredMessageData {
+  content: string;
+  role: string | undefined;
+  additional_kwargs?: AdditionalKwargs;
+}
+
 export interface StoredMessage {
   type: string;
-  role: string | undefined;
-  text: string;
+  data: StoredMessageData;
```

Comment:
I do feel like I'd rather shim this, mark it, then remove the shim when 1.0.0 comes out as a breaking change - I know all this is recent but have seen a few people using it already.

Reply:
@jacoblee93 thoughts on the mapV1MessageToStoredMessage ?

Reply:
I'm for it! Just would add a comment saying to remove when ready to make a breaking change so we don't forget about it